transmogrifier/sources/xml/oaidc.py

```diff
@@ -36,45 +36,34 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
         # citation: uses fallback get_citation() method
 
         # content_type
-        fields["content_type"] = [self.source]
+        fields["content_type"] = self.get_content_type()
```

Comment:
I'd be interested in getting your thoughts on this approach. I opted to write a `get_*()` method  for consistency.